docs: wiki updates for v10 (#1989)
jbsparrow/CyberDropDownloader: CONTRIBUTING.md

#### Line width

120 characters max

The project includes predefined [settings for the zed editor](https://github.com/Cyberdrop-DL/cyberdrop-dl/tree/main/.zed)
with formatting options and a recipe to quickly launch `cyberdrop-dl` with `pdb` (the Python debugger) for testing

### Code formatting with pre-commit hooks

This project uses pre-commit hooks to enforce consistent code style and identify common issues early on.

These hooks run automatically on every `git commit`. If a check fails, the commit is blocked and `ruff` will attempt to

> automatically fix the files. You may need to fix the fix some errors manually if `ruff` can not do it automatically.

Author: ```suggestion
automatically fix the files. You may need to fix some errors manually if `ruff` can not do it automatically.
```

Once everything is fixed, `git add` the changes and commit again.

> [!TIP]

> > You can trigger linting and formatting manually, run `uv run ruff check --fix` then `uv run ruff format`

Author: ```suggestion
> To trigger linting and formatting manually, run `uv run ruff check --fix` then `uv run ruff format`
```
